templates/pdf/android_source_analysis_pdf.html

```diff
@@ -66,6 +66,10 @@ <h3 class="panel-title"><i class="fa fa-clock-o fa-fw"></i> <span class="glyphic
                             </div>
                             <div class="panel-body">
                                 <div class="list-group">
+                                    <p>                                       
+                                        <span class="label label-warning">Name</span>
```

Comment:
Why `warning` ? It should follow the same UI style as other components in the same block.